### DISCUSSION TITLE
How can I enable master password reset option?

### DISCUSSION THREAD

**Discussion by tapashdatta** [dani-garcia/vaultwarden#2884] [Q&A]
One of my user forgot his password. But I was unable to help him to reset his password. I could not find any option either to reset master password from admin portal. Any idea how to do that please? Thanks

**Comment by jjlin** [comment]
There's no way to do this; see https://bitwarden.com/help/forgot-master-password/.

Note that Vaultwarden currently doesn't support admin password reset, and even if it did, you would have to set that up before a user loses their password.
